docs/source/model-registry.rst

```diff
@@ -184,6 +184,52 @@ as shown below to create a new version of the model.
     )
 
 
+Databricks Unity Catalog Model Registry
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To use Databricks unity catalog model registry, set MLflow registry URI to "databricks-uc", and set the two environmental variables "DATABRICKS_HOST" and "DATABRICKS_TOKEN". If you are using Databricks OAuth authentication, set the three environmental variables "DATABRICKS_HOST", "DATABRICKS_CLIENT_ID" and "DATABRICKS_CLIENT_SECRET". If you have configured '~/.databrickscfg' file, then you don't need to set above environmental variables, instead you can set MLflow registry URI to "databricks-uc://{profile}", the '{profile}' part is the profile name in your '~/.databrickscfg' configuration file. For details of Databricks authentication types, please refer to document 'https://docs.databricks.com/en/dev-tools/auth/index.html#unified-auth'.
```

Comment:
```suggestion
To use Databricks unity catalog model registry, set MLflow registry URI to ``"databricks-uc"``, and set the two environmental variables ``"DATABRICKS_HOST"`` and ``"DATABRICKS_TOKEN"``. If you are using Databricks OAuth authentication, set the three environmental variables ``"DATABRICKS_HOST"``, ``"DATABRICKS_CLIENT_ID"`` and ``"DATABRICKS_CLIENT_SECRET"``. If you have configured your access auth within the ``'~/.databrickscfg'`` file, then you don't need to set above environmental variables. Instead, you can set the MLflow registry URI to ``"databricks-uc://{profile}"``, the ``'{profile}'`` part being the profile name in your ``'~/.databrickscfg'`` configuration file. For details of Databricks authentication types, please refer to `this document <https://docs.databricks.com/en/dev-tools/auth/index.html#unified-auth>`.
```
Formatting highlights just to break up the text and make the code-specific parts stand out :)